OfferSpace.Web/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Web;
+using System.Web.Optimization;
+
+namespace OfferSpace.Web
+{
+    public class BundleConfig
+    {
+        // Дополнительные сведения об объединении см. на странице https://go.microsoft.com/fwlink/?LinkId=301862
```

Comment:
Does anybody use Visual Studio with russian language pack? :) if yes - reinstall it and use ONLY english version.